src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -229,7 +229,7 @@ public static IEnumerable<AnalyzerPerformanceInfo> ToAnalyzerPerformanceInfo(thi
             }
 
             Contract.ThrowIfFalse(project.SupportsCompilation);
-            AssertCompilation(project, compilation);
+            //AssertCompilation(project, compilation); //PROTOTYPE: this assert no longer holds
```

Comment:
Just because this isn't properly threading things into the compilation tracker?

Reply:
Pretty much. I started playing with threading it into the tracker and quickly got out of my depth, but I assume thats where it will end up.

---
In reply to: [388644159](https://github.com/dotnet/roslyn/pull/41789#discussion_r388644159) [](ancestors = 388644159)